docarray/array/doc_vec/doc_vec.py

```diff
@@ -156,7 +157,7 @@ class DocVec(AnyDocArray[T_doc]):
         AnyTensor or Union of NdArray and TorchTensor
     """
 
-    doc_type: Type[T_doc]
+    doc_type: Type[T_doc] = AnyDoc  # TODO(johannes): should this be BaseDoc?
```

Comment:
hmmm Maybe yes by default it should be BaseDoc since DocVec cannot be instantiate with AnyDoc